packages/hmr-plugin/src/tests/hmr-manager.spec.ts

```diff
@@ -225,4 +229,47 @@ describe('HMR Plugin', () => {
     // Assert
     expect(MockState.destroy).toEqual(true);
   }));
+
+  it('check if the state is set correctly ', fakeAsync(async () => {
+    const { store } = await hmrTestBed(AppV2MockModule, {
```

Comment:
I would recommend moving the declaration of `AppV2MockModule` into the test, before this line to keep everything together.

Reply:
done